console/web3ext/web3ext.go

```diff
@@ -269,6 +269,16 @@ web3._extend({
 			call: 'admin_sleepBlocks',
 			params: 2
 		}),
+		new web3._extend.Method({
+			name: 'startHTTP',
+			call: 'admin_startHTTP',
+			params: 5,
+			inputFormatter: [null, null, null, null, null]
+		}),
+		new web3._extend.Method({
+			name: 'stopHTTP',
+			call: 'admin_stopHTTP'
+		}),
 		new web3._extend.Method({
```

Comment:
What about adding the comment that startRPC and stopRPC is deprecated as original code does?

Reply:
I added comments. Thanks!